src/toil/jobStores/aws/jobStore.py

```diff
@@ -47,6 +49,47 @@
 
 log = logging.getLogger(__name__)
 
+s3partSize = 2**20*50 
+
+
+def copyKeyMultipart(newKeyName, dstBucketName, srcBucketName, srcKey, headers={}):
```

Comment:
And you can't use instances of mutable types as default arguments.

https://pythonconquerstheuniverse.wordpress.com/2012/02/15/mutable-default-arguments/